DEVELOPER_GUIDE.md

```diff
@@ -14,12 +16,24 @@ Read more about extensibility [here](https://github.com/opensearch-project/OpenS
 
 ## Getting Started
 
-### Git Clone IndependentPlugin Repo
-Fork [IndependentPlugin](https://github.com/owaiskazi19/IndependentPlugin) and clone locally, e.g. `git clone https://github.com/[your username]/IndependentPlugin.git`.
+### Git Clone OpenSearch SDK Repo
+Fork [OpenSearch SDK](https://github.com/opensearch-project/opensearch-sdk) and clone locally, e.g. `git clone https://github.com/[your username]/opensearch-sdk.git`.
 
-### Run Independent Plugin
+### Git Clone OpenSearch Repo
+Fork [OpenSearch](https://github.com/opensearch-project/OpenSearch) and clone locally, e.g. `git clone https://github.com/[your username]/OpenSearch.git`.
 
-Run main script using `gradlew run`.
+## Publish OpenSearch feature/extensions Branch to Maven local
+The work done to support the extensions framework is located on the `feature/extensions` branch of the OpenSearch project. It is necessary to publish the dependencies of this branch to your local maven repository prior to running the OpenSearch SDK on a seperate process. 
+
+- First navigate to the directory that OpenSearch has been cloned to
+- Checkout the correct branch, e.g. `git checkout feature/extensions`.
+- Run `./gradlew publishToMavenLocal`. 
```

Comment:
The next step should be run `./gradlew check` to make sure the build is successful